sqlmesh/core/model/definition.py

```diff
@@ -1736,10 +1746,11 @@ def create_sql_model(
 
     pre_statements = pre_statements or []
     post_statements = post_statements or []
+    audit_expressions = audit_expressions or []
 
     if not python_env:
         python_env = _python_env(
-            [*pre_statements, query, *post_statements],
+            [*pre_statements, query, *post_statements, *audit_expressions],
```

Comment:
is this a breaking change? are we going to need a migration? will this change data hashes?

Reply:
I believe since by scanning the audit expressions and adding its macros to the model's `python_env` will only result in changes when an audit contains macro references, this shouldn't affect the data hash or introduce any breaking changes. as previously, if the user defined a macro within an audit it wouldn't be resolved and would cause the plan to fail